resources/views/components/hidden-controls-toggle-button.blade.php

```diff
@@ -0,0 +1,7 @@
+@props([
+    'label' => 'Moderate', // string
+])
+
+<button id="hidden-controls-toggle-button" class="absolute bottom-2 right-0 mb-2 btn" onclick="toggleHiddenControls()">
+    {{ $label }} ▼
```

Comment:
```suggestion
    {{ $slot }} ▼
```

Using a slot instead of `$label` aligns more closely with how HTML buttons semantically work.

```html
<!-- native button -->
<button>Moderate</button>

<!-- custom button -->
<x-button>Moderate</x-button>

<!-- another custom button -->
<x-hidden-controls-toggle-button>Moderate</x-hidden-controls-toggle-button>
```